ansible/roles/kubernetes-addons/files/common/kube-addon-update.sh

```diff
@@ -262,7 +263,11 @@ function create-object() {
     log INFO "Creating new ${obj_type} from file ${file_path} in namespace ${namespace}, name: ${obj_name}"
     # this will keep on failing if the ${file_path} disappeared in the meantime.
     # Do not use too many retries.
-    run-until-success "${KUBECTL} create --namespace=${namespace} -f ${file_path}" ${NUM_TRIES} ${DELAY_AFTER_ERROR_SEC}
+    if [[ -n "${namespace}" ]]; then
+        run-until-success "${KUBECTL} "${KUBECTL_OPTS}" create --namespace=${namespace} -f ${file_path}" ${NUM_TRIES} ${DELAY_AFTER_ERROR_SEC}
```

Comment:
In this case I find it redundant as the namespace is parsed from the file anyway.
Though, I will keep it in it so it is 1:1 cherry pick.